ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/gio-form-json-schema.component.ts

```diff
@@ -101,6 +103,8 @@ export class GioFormJsonSchemaComponent implements ControlValueAccessor, OnInit,
     this.formGroup.valueChanges
       .pipe(
         takeUntil(this.unsubscribe$),
+        // Group all valueChanges events emit by formly in a single one
+        debounceTime(50),
```

Comment:
500 must be enough WDYT ?
```suggestion
        debounceTime(500),
```

Reply:
I once heard that human eye is not able to detect something under 500ms. So I think 200 should be enough. But it's only a suggestion if you want to keep 50, I'm fine with it too.